data-products/src/main/scala/org/sunbird/analytics/exhaust/collection/ProgressExhaustJob.scala

```diff
@@ -108,7 +108,14 @@ object ProgressExhaustJob extends optional.Application with BaseCollectionExhaus
     import spark.implicits._
     val contentDataDF = hierarchyData.rdd.map(row => {
       val hierarchy = JSONUtils.deserialize[Map[String, AnyRef]](row.getString(1))
-      filterAssessmentsFromHierarchy(List(hierarchy), List(AppConf.getConfig("assessment.metrics.supported.contenttype")), AssessmentData(row.getString(0), List()))
+
+      val assessmentFilters = Map(
+        "assessmentTypes" -> AppConf.getConfig("assessment.metrics.supported.contenttype").split(",").toList,
+        "questionTypes" -> Option(AppConf.getConfig("assessment.metrics.supported.objecttype")).getOrElse("QuestionSet").split(",").toList,
+        "primaryCategories" -> AppConf.getConfig("assessment.metrics.supported.primaryCategories").split(",").toList
```

Comment:
Raise a configuration PR in data pipeline repo for this configurations

Reply:
@reshmi-nair : Sure